lib/virt_autotest/utils.pm

```diff
@@ -105,23 +105,48 @@ sub is_xen_host {
     return get_var("XEN") || check_var("SYSTEM_ROLE", "xen") || check_var("HOST_HYPERVISOR", "xen") || check_var("REGRESSION", "xen-hypervisor");
 }
 
-#check host to make sure it works well
+#check kernels by grep keywords from journals
 #welcome everybody to extend this function
-sub check_host {
+sub check_failures_in_journal {
+    my $machine //= 'localhost';
+
+    my $all_failures = "";
+    my $bug = get_var('BUG_NUMBER', '55555');
+    my @warnings = ('Started Process Core Dump', 'Call Trace');
```

Comment:
I am afraid "Call Trace" is too general. Even warning message will contain "Call Trace". So tons of tests will fail on a minor issue.